tests/e2e/cluster_downgrade_test.go

```diff
@@ -67,7 +69,7 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 	e2e.BeforeTest(t)
 
 	t.Logf("Create cluster with version %s", currentVersionStr)
-	epc := newCluster(t, clusterSize)
+	epc := newCluster(t, clusterSize, 10, t.TempDir())
```

Comment:
The dataDirPath is t.TempDir by default, FYI. https://github.com/etcd-io/etcd/blob/61736c329d19cadfd0bef3c3ee6af63ee010d9ba/tests/framework/e2e/cluster.go#L504
```suggestion
	epc := newCluster(t, clusterSize, 10)
```

Reply:
And also the hard coded value 10 is used multiple times, can you define a constant?

Reply:
updating.